### PR TITLE
Add szshaxx to the legacy exploits page

### PR DESCRIPTION
**Pull request by pumpkintheproot**
**Description**
This pull request adds a new exploit triggered by overflowing Mario Kart Wii's output buffer, resulting in the ability to execute arbitary code. The Github page for the exploit can be found [here](https://github.com/MikeIsAStar/szsHaxx/tree/v1.0.0.0)

New exploit basically. Easier but requires an SD card with less than 2GB of capacity.

### CODE REVIEW

File: _pages/en_US/szshaxx.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+6. Make sure `WiiConnect24` is enabled on your Wii before performing the exploit
+
+    ![](/images/exploits/szshaxx/wiiconnect24.png))
+
+7. Delete any save data that exists for Mario Kart Wii. 
+
+    ![](/images/exploits/szshaxx/deleted.png))
+
+    If there is data you would like to keep, we would recommend moving the data to a seperate SD card first.
```
Comment (Naim2000) [inline _pages/en_US/szshaxx.md:49]: Can't copy the save data for Mario kart Wii

**Review by Naim2000 — COMMENTED** [review]
Not to hate but I don't think this is really worth adding to the guide. It's cool but ultimately superseded by Wilbrand
Perhaps it would be a decent last resort if you could do it without accessing Wii settings. (maybe you can if you accept the EULA through Wii Shop Channel? That automatically enables WiiConnect24)

Also, there's a mouse cursor in one of your screenshots.

**Comment by pumpkintheproot** [comment]
> Not to hate but I don't think this is really worth adding to the guide. It's cool but ultimately superseded by Wilbrand Perhaps it would be a decent last resort if you could do it without accessing Wii settings. (maybe you can if you accept the EULA through Wii Shop Channel? That automatically enables WiiConnect24)
> 
> Also, there's a mouse cursor in one of your screenshots.

it's just a method that could be considered easier for some 
also yeah,  ill redo that screenshot as i realised last moment

**Comment by Ingunar** [comment]
Why is there szshaxx.md?
There is no need for it.

This is an alternative/legacy exploit.

File: _pages/en_US/legacy-exploits.md [hunk 2]
```diff
@@ -69,6 +69,15 @@ Note for vWii: if one of these exploits is planned to be used, it is recommended
 + Superceded mostly by Letterbomb, Bluebomb and the Compat Title Installer.
 + [Wiibrew Page](https://wiibrew.org/wiki/Yu-Gi-Vah)
 
+#### szsHaxx
```
Comment (playerjmr) [inline _pages/en_US/legacy-exploits.md:72]:
```suggestion
#### szsHaxx (Wii only)
```

**Pull request merged**